### DISCUSSION TITLE
请问“查看权限设置”如何关闭？目前没找到好的办法，隔段时间通知栏会出现一次，挺讨厌的

### DISCUSSION THREAD

**Discussion by 1tonsilla** [gkd-kit/gkd#744] [General]
![83FD60EED70DCED2C0D0C31A61893653](https://e.gkd.li/f33eac92-2c06-4fb4-8609-74690e079148)
我是vivo系统

**Comment by lisonge** [comment]
长按通知-更多设置-关闭这个系统应用的通知权限

**Reply by 1tonsilla** [reply]
这是Android系统 通知，和普通app不一样，无法直接关闭；我刚又划掉啦，等下次再出现时候截图出来，大佬就懂了~

**Reply by lisonge** [reply]
我是小米手机，我记得是通过类似我上面说的方案关闭了这个通知

所以我这边没有这个通知了，我已经忘记如何关闭了

**Reply by lisonge** [reply]
试试这个版本呢，我看了一下那个警告通知是没有在清单文件里声明某个属性导致的，下面这个版本加上了这个属性

https://github.com/gkd-kit/gkd/actions/runs/11203447766/artifacts/2020967107

**Reply by 1tonsilla** [reply]
> 试试这个版本呢，我看了一下那个警告通知是没有在清单文件里声明某个属性导致的，下面这个版本加上了这个属性
> 
> https://github.com/gkd-kit/gkd/actions/runs/11203447766/artifacts/2020967107

好的~感谢大佬，我使用，然后观察观察

**Reply by 1tonsilla** [reply]
> 试试这个版本呢，我看了一下那个警告通知是没有在清单文件里声明某个属性导致的，下面这个版本加上了这个属性
> 
> https://github.com/gkd-kit/gkd/actions/runs/11203447766/artifacts/2020967107

应该就是你说的这个原因，我用了两三个星期了 ，没有再出现这个提示！谢谢！

**Reply by lisonge** [reply]
可以使用最新的 beta 版本，这个修复已经在最新测试版中

**Reply by lisonge** [reply]
https://github.com/gkd-kit/gkd/releases

**Reply by 1tonsilla** [reply]
> https://github.com/gkd-kit/gkd/releases

收到，幸苦了！
